### ISSUE TITLE
HTTP/* keyword is deprecated

### ISSUE THREAD

**Issue by 7hens** [pfeiferj/vscode-hurl#2]
## Environment

```
hurl 3.0.0 libcurl/7.79.1 (SecureTransport) LibreSSL/3.3.5 zlib/1.2.11 nghttp2/1.45.1
Features (libcurl):  alt-svc AsynchDNS HSTS HTTP2 IPv6 Largefile libz NTLM NTLM_WB SPNEGO SSL UnixSockets
Features (built-in): brotli
```

## Description

Syntax highlighting for hurl files only works with keywords "HTTP/* 200", using "HTTP 200" breaks syntax highlighting. 

But when executing  the hurl command for the file using "HTTP/* 200", there is a warning:

```
The 'HTTP/*' keyword is deprecated, please use 'HTTP' instead
```

**Comment by pfeiferj** [comment]
Sorry for the delayed response, planning on putting together a fix tomorrow.

**Comment by pfeiferj** [comment]
Resolved in v1.2.0
